commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/ram/RamFileSystem.java

```diff
@@ -193,9 +193,9 @@ public void importTree(final File file) throws FileSystemException {
     /**
      * Import the given file with the name relative to the given root
      *
-     * @param fo
-     * @param root
-     * @throws FileSystemException
+     * @param fo the source {@link FileObject} file to import
```

Comment:
Sentences end with a period, which did nicely elsewhere in this PR.

Reply:
Yep. Changed